### PR TITLE
List items need to repair

### PR DESCRIPTION
**Pull request by radvc**
Reference for this issue https://github.com/ReadyResponder/ReadyResponder/issues/405. In Inventory I display repairs that are not completed with table.

### CODE REVIEW

File: app/views/item_categories/index.html.erb [hunk 1]
```diff
@@ -18,3 +18,34 @@
...
+    <tbody>
+      <% @repairs.each do |repair|%>
+        <tr>
+          <td><%= link_to repair.item.name, item_path(repair.item) %></td>
+          <td><%= repair.service_date%></td>
+          <td><%= repair.status %></td>
+          <td><%= repair.description %></td>
+          <td><%= repair.repairer_name %></td>
+          <td><%= repair.cost %></td>
+          <td><%= table_button_link 'Edit', edit_repair_path(repair) %></td>
```
Comment (kgf) [inline app/views/item_categories/index.html.erb:46]: This is nice work. Could you add the conditional on this line? Not everyone will be able to edit a repair. You can find the correct format on person index.

**Comment by kgf** [comment]
@radvc thanks for your help. Would you be willing to add a test or two? Perhaps a unit test on the new scope, or a feature spec for the view.

**Comment by radvc** [comment]
@kgf okay no problem, I will add test

**Comment by kgf** [comment]
Great, @radvc . And the conditional on the edit button? ( Just checking that you saw that.)

**Comment by radvc** [comment]
@kgf conditional on edit button?

**Comment by kgf** [comment]
Thanks for your help, @radvc 
Kevin

**Pull request merged**